src/services/authentication/hooks/index.js

```diff
@@ -35,8 +36,8 @@ const bruteForceCheck = async (context) => {
 				if (timeDifference < allowedTimeDifference) {
 					throw new TooManyRequests(
 						'Brute Force Prevention!', {
-							timeToWait: allowedTimeDifference - Math.ceil(timeDifference),
-						},
+						timeToWait: allowedTimeDifference - Math.ceil(timeDifference),
+					},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 tabs but found 5.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5091261)